swagger_sdk_gen_v3.sh

```diff
@@ -3,12 +3,12 @@ if $Build_Codegen ; then
 fi
 if [ "$Branch" = "snapshot" ]
   then
-    url="http://newapi-nightly.capillary.in/tl-docs/v2/api-docs"
-    version="https://nightly.capillary.in/v3/meta/version"
+    url="http://crm-nightly-new.cc.capillarytech.com/tl-docs/v2/api-docs"
+    version="https://crm-nightly-new.cc.capillarytech.com/v3/meta/version"
 elif [ "$Branch" = "production" ]
   then
-    url="http://newapi-nightly.capillary.in/tl-docs/v2/api-docs"
-    version="https://nightly.capillary.in/v3/meta/version"
+    url="http://crm-nightly-new.cc.capillarytech.com/tl-docs/v2/api-docs"
```

Comment:
this is production. this should point to staging. existing url is also wrong. please keep the staging url here.